File: Lab1/Lab1.cpp
//Задано сторону квадрата. Знайти його діагональ, периметр та площу.
#include <iostream>
#include <math.h> 

using std::cout;
using std::cin;
using std::endl;

int main()
{
    setlocale(LC_ALL, "Russian");
    // Об'ява змінних
    float Size;
    float Perimeters;
    float Diagonal;
    float Area;
    // Введення сторони
    cout << "Input the side of square: ";
    cin  >> Size;
    // Обчислення виводу
    Perimeters = Size * 4; // Периметр
    Diagonal = Size * sqrt(2); // Діагональ
    Area = Size * Size; // Площа
    // Вивід 
    cout << "Perimeters:\t" << Perimeters << endl;
    cout << "Diagonal:\t" << Diagonal << endl;
    cout << "Area:\t\t" << Area << endl;
    return 0;
}

File: Lab1.py
# Задано сторону квадрата. Знайти його діагональ, периметр та площу.
import math

#Ввід даних
side = float(input('Input the side of square: '))  # Ввід сторони квадрату

perimeter = 4 * side            # Обчислення периметра
diagonal = math.sqrt(2) * side  # Обчислення діагоналі
area = side ** 2                # Обчислення площі

# Вивід обчислень на екран
print('Perimeter is ', perimeter)
print('Diagonal is ', diagonal)
print('Area is ', area)
